packages/core/carbon.yml

```diff
@@ -0,0 +1,527 @@
+  parallel-coordinates:
+    name: Parallel coordinates
+    status: draft
+    type: component
+    tags:
+      - data-visualization
+      - correlation
+    framework: design-only
+    platform: web
+    thumbnailPath: /src/thumbnails/parallel.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/complex-charts#parallel-coordinates
+  pie:
+    name: Pie
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+      - part-to-whole
+    framework: vanilla
+    platform: web
+    thumbnailPath: /src/thumbnails/pie.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#pie
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-pie
+  radar:
+    name: Radar
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+      - comparison
+    framework: vanilla
+    platform: web
+    thumbnailPath: /src/thumbnails/radar.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#radar
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-radar
+  scatter:
+    name: Scatter
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+      - correlation
+    framework: vanilla
+    platform: web
+    thumbnailPath: /src/thumbnails/scatter.svg
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/simple-charts#scatter
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-scatter
+  sparkline:
+    name: Sparkline
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+    framework: vanilla
+    platform: web
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/getting-started
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-sparkline
+  step:
+    name: Step
+    status: stable
+    type: component
+    tags:
+      - data-visualization
+    framework: vanilla
+    platform: web
+    externalDocsUrl: https://www.carbondesignsystem.com/data-visualization/getting-started
+    demoLinks:
+      - type: storybook
+        name: Storybook
+        action: link
+        url: https://charts.carbondesignsystem.com/?path=/story/simple-charts-step
+  stream:
+    name: Stream
+    status: draft
+    type: component
+    tags:
+      - data-visualization
+      - trend
+    framework: design-only
+    platform: web
+    thumbnailPath: /src/thumbnails/stream.svg
+    noIndex: true
```

Comment:
I put Steam as `noIndex` so it doesn't show up in the catalogs. It looks grayed out in the legacy site with no click target. This right or wrong?

Reply:
yeah better to exclude for now